doc/src/BCMPackage.rst

```diff
@@ -18,10 +23,17 @@ bcm_package
 
 This setups a non-boost package in cmake. This is similar to ``bcm_boost_package`` but with extra flexibility as it does not have boost specific conventions. This function creates a library target that will be installed and exported. In addition, the target will be setup to auto-link for the tests.
 
+.. "setup to auto-link for the tests". This is a bit unclear. Maybe say
+   "Subsequent uses of 'bcm_test' will automatically include this target as a
+   dependency" or something along these lines.
```

Comment:
Yes I think that sounds better. Although "subsequent uses" is not needed. If you write:

```cmake
bcm_test(...)
bcm_package(...)
```

This is because `bcm_test` links against the libraries or package at generation time.